Amazon.IonDotnet.Tests/Internals/ReaderLocalTableTest.cs

```diff
@@ -48,5 +56,242 @@ void assertTable(ISymbolTable tab, params (string sym, int id)[] syms)
                 ("c", 12)
             );
         }
+
+        [TestMethod]
+        public void TestLocalSymbolTableAppend()
+        {
+            string text = LocalSymbolTablePrefix +
```

Comment:
Maybe a non-issue, but should we use var instead?

Unless unit testing makes sense to be specific for types.

Reply:
Updated to var where it is inferred